src/content/docs/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro.mdx

```diff
@@ -105,12 +105,11 @@ with New Relic.
 
     Onec your app is instrumented and configured to export data to New Relic, you should be able to find your data in the New Relic UI:
 
-    * Find your entity at <DNT>**All entities -> Services - OpenTelemetry**</DNT>. The entity name is set to the value of the app's `service.name` resource attribute. For more information on how New Relic service entities are derived from OpenTelemetry resource attributes, see [Services](/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources/#services)
+    * Find your entity at <DNT>**All entities -> Services - OpenTelemetry**</DNT>. The entity name is set to the value of the app's `service.name` resource attribute. For more information on how New Relic service entities are derived from OpenTelemetry resource attributes, see [Services](/docs/opentelemetry/best-practices/opentelemetry-best-practices-resources/#services).
```

Comment:
I think we use > for paths: `<DNT>**All entities > Services - OpenTelemetry**</DNT>`